ckanext/switzerland/helpers/plugin_utils.py

```diff
@@ -319,11 +323,21 @@ def _transform_package_dates(pkg_dict):
 
 
 def _transform_datetime_to_isoformat(value):
-    """derive isoformat from datepicker date format"""
+    """Transform dates in ckanext.switzerland.date_picker_format to isodates.
+    If dates are already in isoformat, just return them.
```

Comment:
I don't think that case ever occurs that isodates enter that function. As far as I could tell all dates enter it in the German date format as this is now the output date format returned by the date validator. 

But since this extra case in the transformation does not do any harm, it will be also okay for me to merge it as it is. I am just curious: did you observe a bug there and would it be reproducible?